documentation/configuration.md

```diff
@@ -295,10 +324,19 @@ Unlike the other diagnostic artifacts (for example, traces), memory dumps aren't
 </details>
 
 <details>
-  <summary>Environment Variables</summary>
+  <summary>ConfigMap</summary>
+  
+  ```yaml
+  DotnetMonitor_Storage__DumpTempFolder: "/ephemeral-directory/"
```

Comment:
I know this was the value before, but let's give it a real example. Maybe `/diag/dumps/`.